lib/teleterm/api/proto/v1/cluster.proto

```diff
@@ -22,15 +22,21 @@ option go_package = "github.com/gravitational/teleport/lib/teleterm/v1";
 message Cluster {
     // uri is the cluster resource URI
     string uri = 1;
-    // name is the cluster name
+    // name is the host/profile name
     string name = 2;
+    // actual name of the cluster that can be different than the host/profile name
+    // it is a temporary solution, the plan is to replace `name` field with `actual_name` value
+    // (and use it in the generated `uri`)
+    string actual_name = 3;
+    // proxy address (only for root clusters)
+    string proxy_host = 4;
     // connected indicates if connection to the cluster can be established, that is if we have a
     // cert for the cluster that hasn't expired
-    bool connected = 3;
+    bool connected = 5;
```

Comment:
I'd add them at the end as new fields instead of reordering existing ones, but at this point it doesn't matter much. I don't even know if we'll have to care about backwards compatibility of those protobuf messages going forward.

Reply:
I will keep that in mind for the future, but IMO we don't to have to care about the compatibility here